Utils/github_workflow_scripts/autobump_rn.py

```diff
@@ -0,0 +1,1170 @@
+            # todo: delete it, only for testing
+            return "Pack MyPack version was automatically bumped to 1.0.2."
+        with checkout(self.git_repo, self.branch):
+            for pack_auto_bumper in self.packs_to_autobump:
+                pack_auto_bumper.set_pr_changed_rn_related_data()
+            self.git_repo.git.merge(
+                f"origin/{BASE}", "-Xtheirs", "-m", MERGE_FROM_MASTER_COMMIT_MESSAGE
+            )
+            for pack_auto_bumper in self.packs_to_autobump:
+                new_version = pack_auto_bumper.autobump()
+                print(f"Pack {pack_auto_bumper.pack_id} new version: {new_version}.")
+                self.git_repo.git.add(f"{PACKS_DIR}/{pack_auto_bumper.pack_id}")
+                self.git_repo.git.commit(
+                    "-m",
+                    COMMIT_MESSAGE.format(
+                        new_version,
+                        pack_auto_bumper.pack_id,
+                    ),
+                )
+                body += PR_COMMENT.format(
+                    pack_auto_bumper.pack_id,
+                    new_version,
+                )
+            print(f"[{self.pr.number}] Committed the changes. Commenting on the pr: \n{body}.\n")
+            self.git_repo.git.push()
+            self.pr.create_issue_comment(body)
+        return body
+
+
+class AutoBumperManager:
+    def __init__(
+        self,
+        github_repo_obj: Repository,
+        git_repo_obj: Repo,
+        run_id: str,
+    ):
+        """
+        Args:
+            github_repo_obj: GitHub API repo object.
+            git_repo_obj: Git API repo object.
+            run_id: GitHub action run id.
+        """
+        self.github_repo_obj = github_repo_obj
+        self.git_repo_obj = git_repo_obj
+        self.run_id = run_id
+
+    def manage(self):
+        """
+        Iterates over all PR's in the repo, checks if all conditions to update pack version met.
+        If no - skips checks for the pack/branch.
+        If the pack meets all conditions to autobump pack version, it bumps the version.
+        """
+        for pr in self.github_repo_obj.get_pulls(
+            state="open", sort="created", base=BASE
+        ):
+            print(
+                f"{t.yellow}Looking on pr number [{pr.number}]: last updated: "
+                f"{str(pr.updated_at)}, branch={pr.head.ref}"
+            )
+
+            conditions = [
+                LastModifiedCondition(pr=pr, git_repo=self.git_repo_obj),
+                LabelCondition(pr=pr, git_repo=self.git_repo_obj),
+                AddedRNFilesCondition(pr=pr, git_repo=self.git_repo_obj),
+                HasConflictOnAllowedFilesCondition(pr=pr, git_repo=self.git_repo_obj),
+            ]
+            for c1, c2 in pairwise(conditions):
+                c1.set_next_condition(c2)
+
+            base_cond_result = conditions[0].check()
+            if base_cond_result.should_skip:
+                continue
```

Comment:
please log skips

Reply:
The skip logs are printed in the BaseCondition.check() function.